### ISSUE TITLE
How to add newly created category description field in REST API response

### ISSUE THREAD

**Issue by davidverbeek** [akeneo/pim-community-dev#11057]
Hello All,

I have created new category description field using below reference url.

https://docs.akeneo.com/3.2/manipulate_pim_data/category/add_new_properties_to_a_category.html#add-non-translatable-properties-to-your-own-category

Now I want this description field to be present when the client application calls 
"/api/rest/v1/categories" akeneo rest api.

Currenlty when "/api/rest/v1/categories" api is called it returns below response,

{
        "_links": {
          "self": {
            "href": "https://demo.akeneo.com/api/rest/v1/categories/woman"
          }
        },
        "code": "woman",
        "parent": "winter_collection",
        "labels": {
          "en_US": "Woman",
          "fr_FR": "Femme",
          "de_DE": "Damen"
        }
      }

I want new field "description" somewhere in the response so that my client application can get category description field along with category name.

Basically I have to alter the rest api category response.

Please help.

**Comment by Doodoune** [comment]
Hi @davidverbeek,
I think you have to add it in the CategoryNormalizer.
Regards,

**Comment by jmleroux** [comment]
I close this PR without any activity since several month.
Feel free to reopen it if needed.

Regards
